[32m2021-07-23 16:15:16[0m - dagster - [34mDEBUG[0m - cumulus_pipeline - 94546498-8f5b-4ec9-8fb8-fca85bb9794d - 945 - format_date - LOGS_CAPTURED - Started capturing logs for solid: format_date.
[32m2021-07-23 16:15:16[0m - dagster - [34mDEBUG[0m - cumulus_pipeline - 94546498-8f5b-4ec9-8fb8-fca85bb9794d - 945 - format_date - STEP_START - Started execution of step "format_date".
[32m2021-07-23 16:15:16[0m - dagster - [34mDEBUG[0m - cumulus_pipeline - 94546498-8f5b-4ec9-8fb8-fca85bb9794d - 945 - format_date - LOADED_INPUT - Loaded input "df" using input manager "io_manager", from output "result" of step "extract_dimensions"
[32m2021-07-23 16:15:16[0m - dagster - [34mDEBUG[0m - cumulus_pipeline - 94546498-8f5b-4ec9-8fb8-fca85bb9794d - 945 - format_date - STEP_INPUT - Got input "df" of type "Any". (Type check passed).
/home/pitanga/.local/share/virtualenvs/situated-views-etl-KhhASCmw/lib/python3.6/site-packages/pandas/core/indexing.py:1743: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  isetter(ilocs[0], value)
[32m2021-07-23 16:15:18[0m - dagster - [34mDEBUG[0m - cumulus_pipeline - 94546498-8f5b-4ec9-8fb8-fca85bb9794d - 945 - format_date - STEP_OUTPUT - Yielded output "result" of type "Any". (Type check passed).
[32m2021-07-23 16:15:19[0m - dagster - [34mDEBUG[0m - cumulus_pipeline - 94546498-8f5b-4ec9-8fb8-fca85bb9794d - 945 - format_date - HANDLED_OUTPUT - Handled output "result" using IO manager "io_manager"
[32m2021-07-23 16:15:19[0m - dagster - [34mDEBUG[0m - cumulus_pipeline - 94546498-8f5b-4ec9-8fb8-fca85bb9794d - 945 - format_date - STEP_SUCCESS - Finished execution of step "format_date" in 2.14s.
